src/javascript/gameInstance.js

```diff
@@ -417,20 +441,22 @@ export default class GameInstance {
           : this.audio.getCurrentTime();
       if (!this.vm.started) return cTime;
     }
-    this.playTime = cTime + this.vm.noteSpeedInSec;
+    this.playTime = cTime + this.noteDelay;
     this.currentTime = cTime;
     return cTime;
   }
 
   getNoteTiming() {
-    return this.vm.playMode ? this.playTime : this.currentTime; // in editor mode, time without note drop delay is used.
+    return this.playTime;
   }
```

Comment:
If this method is only used within the object to maintain internal state, don't add another layer of indirection. If this method is also part of the public API, then disregard this comment.

Reply:
Make sense, done!